eclass/linux-mod-r1.eclass

```diff
@@ -330,6 +331,15 @@ linux-mod-r1_pkg_setup() {
 	debug-print-function ${FUNCNAME[0]} "${@}"
 	[[ ${MERGE_TYPE} != binary ]] || return 0
 	_MODULES_GLOBAL[ran:pkg_setup]=1
+
+	if [[ -z ${ROOT} && ${MODULES_INITRAMFS_IUSE} ]] &&
+		use dist-kernel && use ${MODULES_INITRAMFS_IUSE#+}
+	then
+		# Check, but don't die because we can fix the problem and then
+		# emerge --config ... to re-run installation.
+		nonfatal mount-boot_check_status
```

Comment:
Issues aside, does feel like adding this is not super important though. Most people won't notice it in setup vs pretend, and modules tend to have short build times so will reach pkg_postinst and fail before you realize.

May not be worth changing things around, not that I mind if really want this.

Reply:
> Issues aside, does feel like adding this is not super important though. Most people won't notice it in setup vs pretend, and modules tend to have short build times so will reach pkg_postinst and fail before you realize.

Yes, the important part is the adjustment to `dist-kernel-utils.eclass` that makes `dist-kernel_reinstall_initramfs` do this same check.